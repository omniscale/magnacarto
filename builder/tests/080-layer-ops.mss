#labels {
  text-name: [name];
  text-size: 8;

  // test layer based properties, only work for Mapnik
  opacity: 50%;
  comp-op: screen;
  

  image-filters: "invert(), color-blind-deuteranope()";
  direct-image-filters: "invert()";
}
