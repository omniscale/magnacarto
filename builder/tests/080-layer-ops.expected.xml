<Map srs="epsg:3857">
  <Parameters></Parameters>
  <Style name="labels" filter-mode="first" comp-op="screen" opacity="50" image-filters="invert(), color-blind-deuteranope()" direct-image-filters="invert()">
    <Rule>
      <TextSymbolizer size="8">[name]</TextSymbolizer>
    </Rule>
  </Style>
  <Layer name="labels" srs="" status="off">
    <StyleName>labels</StyleName>
  </Layer>
</Map>